Substrate.NetApi/SubstrateClient.cs

```diff
@@ -270,7 +289,38 @@ public async Task ConnectAsync(bool useMetaData, bool standardSubstrate, Cancell
                 }
             }
 
-            //_jsonRpc.TraceSource.Switch.Level = SourceLevels.All;
+            // Start a background task to monitor the WebSocket connection
+            _ = Task.Run(async () =>
```

Comment:
In general we should avoid never-ending task loops. Additionally I suggest to see if there are events that we can use from the va-streamjsonrpc or from the websocket it self.

Reply:
Removed :)